recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  fn: plasmapy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 2a2bbc064a8b5726bc32447973babba9355e5c7d9483b1eebc82c6e83a9a173e
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True    # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - cython
+    - setuptools
+    - numpy
+    - scipy
+    - astropy
+  run:
+    - python
+    - numpy
+    - scipy
+    - astropy
+    - pytest
+    - mpmath
+    - h5py
+    - lmfit
+    - matplotlib
+    - colorama
+    - pytest-astropy
+
+test:
+  imports:
+    - plasmapy
+    - plasmapy.atomic
+    - plasmapy.atomic.tests
+    - plasmapy.classes
+    - plasmapy.classes.tests
+    - plasmapy.constants
+    - plasmapy.io
+    - plasmapy.mathematics
+    - plasmapy.mathematics.tests
+    - plasmapy.physics
+    - plasmapy.physics.tests
+    - plasmapy.utils
+    - plasmapy.utils.tests
+    - plasmapy.visualization
+    - plasmapy.visualization.tests
+
+about:
+  home: https://github.com/PlasmaPy/PlasmaPy
+  license: BSD-3-Clause
+  license_family: BSD
+  #license_file: LICENSE.md
```

Comment:
If the ``license`` is not being packaged you can pack it along with the recipe. After that open a PR to include the license

Reply:
I added it directly in 
9980325 .